log.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 VMware, Inc. or its affiliates. All Rights Reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package amqp091
+
+import (
+	"log"
+	"os"
+)
+
+var ErrLog = log.New(os.Stderr, "[ERROR] amqp091-go: ", log.LstdFlags|log.Lshortfile)
+var WarnLog = log.New(os.Stderr, "[WARNING] amqp091-go: ", log.LstdFlags|log.Lshortfile)
+
+func SetErrorLogger(errlog *log.Logger) {
+	ErrLog = errlog
+}
+
+func SetWarningLogger(warnlog *log.Logger) {
+	WarnLog = warnlog
+}
```

Comment:
Changing the logger should be protected by a Lock or alternatively it needs to be documented that changing the logger is only safe when no clients exist